compiler/rustc_passes/src/dead.rs

```diff
@@ -1131,7 +1132,8 @@ impl<'tcx> DeadVisitor<'tcx> {
         if dead_codes.is_empty() {
             return;
         }
-        dead_codes.sort_by_key(|v| v.level);
+        // FIXME: `dead_codes` should probably be morally equivalent to `IndexMap<(Level, LintExpectationId), (DefId, Symbol)>`
+        dead_codes.sort_by(|a, b| a.level.0.partial_cmp(&b.level.0).unwrap());
```

Comment:
Wait, is `level.0` now `Ord`? Could we change this back to sorting by key?

Reply:
oops yea